programs/clearing_house/src/controller/pnl.rs

```diff
@@ -29,6 +29,8 @@ pub fn settle_pnl(
     oracle_map: &mut OracleMap,
     now: i64,
 ) -> ClearingHouseResult {
+    validate!(!user.bankrupt, ErrorCode::UserBankrupt)?;
```

Comment:
curious when `meets_maintenance_margin_requirement` would not catch this?

Reply:
it would catch it, was just adding it to the all ix that should be blocked